my-app/src/modules/APImanager.js

```diff
@@ -0,0 +1,57 @@
+const remoteURL = "http://localhost:3000"
+
+export default {
+    get(id) {
+        // if isAuthenticated() {
+      return fetch(`${remoteURL}/resource/${id}`, {
```

Comment:
there is no /resource resource 
`getAll(resource) {
        // if (isAuthenticated()) {
            return fetch(`${remoteURL}/${resource}`, {
                "method": "GET",
                "headers": {
                    "Accept": "application/json",
                    "Authorization": `Token ${localStorage.getItem("bangazon_token")}`
                }
            })
                .then(response => response.json())
        // }
    },`